extension/data/modules/usernotes.js

```diff
@@ -1008,7 +1008,7 @@ function usernotes () {
                         <div class="tb-un-note-details">
                             <a class="tb-un-notedelete tb-icons tb-icons-negative" data-note="${key}" data-user="${user.name}" href="javascript:;">${TBui.icons.delete}</a>
                             <span class="note">
-                                <span class="note-type">[${color.text}]</span>
+                                <span class="note-type" ${color.key !== 'none' ? `style="color:${color.color}"` : ''}>[${color.text}]</span>
```

Comment:
I know we don't do this in any of the surrounding code right now, but let's HTML encode this value just to be safe:
```suggestion
                                <span class="note-type" ${color.key !== 'none' ? `style="color:${TBHelpers.htmlEncode(color.color)}"` : ''}>[${color.text}]</span>
```

Reply:
Ah yes, I totally forgot about that. I'll update it soon

Reply:
Good call, though we don't need to worry about injection much here as the subreddit config already has been put through domPurify here: https://github.com/toolbox-team/reddit-moderator-toolbox/blob/master/extension/data/tbcore.js#L1361

Still, if for some reason we find ourselves with html that is technically safe it could still mess up rendering.